lib/web_ui/test/text_editing_test.dart

```diff
@@ -711,6 +703,47 @@ void main() {
       // Confirm that [HybridTextEditing] didn't send any more messages.
       expect(spy.messages, isEmpty);
     });
+
+    test('sets correct input type', () {
+      // Maps flutter input types to DOM inputmode values.
+      const Map<String, String> inputTypesMap = <String, String>{
+        'text': 'text',
+        'number': 'numeric',
+        'phone': 'tel',
+        'emailAddress': 'email',
+        'url': 'url',
+      };
+      int clientId = 0;
+
+      void testAllTypes() {
+        for (String flutterInputType in inputTypesMap.keys) {
```

Comment:
suggestion: we can remove the for loop. For loops usually makes trouble shooting the broken tests harder.

Reply:
Very good suggestion. I removed the for loop and changed it so that all the `expect` calls are directly located in the test function.

The tests are failing for some reason so I'm investigating. I'll push my commit once the test is fixed.